### DISCUSSION TITLE
ImportBundle from Clockify - Activities linked to Projects (not global)

### DISCUSSION THREAD

**Discussion by acseven** [kimai/kimai#4532] [Plugins]
Hi,
The ImportBundle plugin imports Activities as `global`. However, I'm handling a migration of years of Clockify data where all activities are `project specific`:

From the [documentation](https://www.kimai.org/documentation/activity.html): 
![image](https://github.com/kimai/kimai/assets/1176371/36740a7d-3523-4919-a583-7ec5bc23815f)

Is there a way to control how the activities are inserted in the database?

Thanks

**Comment by acseven** [comment]
Nevermind, missed it.

![image](https://github.com/kimai/kimai/assets/1176371/d9b910b7-b953-43cf-aae8-eae6f4285387)

**Comment by kevinpapst** [comment]
Oh interesting. Can you share some results?

I has one customer who migrated about 20k entries. This took a couple of approaches, before it worked, but the plugin wasn't there at that time. 

Wondering if it works a little better today.

To be safe, you should create backups before the import, try, re-import the backup and restart

**Reply by acseven** [reply]
I have some issues to tackle and I'm still testing batches of data, but I will try to share as much as possible. 

One very significant issue is the limitation of file size (1024kb) and max lines (1000), which makes it very time consuming to process.
